README.md

```diff
@@ -1,2 +1,21 @@
 # hello_microverse_project
-Setting up a "Hello Microverse" project with the correct and professional structure.
+
+hello_microverse_project is a basic Microverse project to familiarize with GitHub workflow and create professional repositories.
+
+## Tech Stack
+
+Client
+- HTML
+- CSS
+
+## Key Features
+
+- Create a basic project structure
+- Develop a professional repository
+- Add useful commits
+
+## Authors
+
+Jorge Abrahan Siguenza
+- GitHub: [@jorgeabrahan](https://www.github.com/jorgeabrahan)
+- Linkedin: [@Jorge Abrahan](https://www.linkedin.com/in/jorge-abrahan-siguenza-amaya-a975741b8)
```

Comment:
- I think it is important to use the newly approved microverse README template for your projects. Kindly update your version with the new one 🤗 

Here is a link to the [newly updated template](https://github.com/microverseinc/readme-template/blob/master/README.md)

⚠️ Kindly ensure to add all information regarding the "hello microverse" project in the Readme as well.